home-manager/programs/waybar/style.css

```
@@ -1,166 +1,140 @@
-@define-color bg_hover rgba(200, 200, 200, 0.3);
-/* base background color of active elements */
-@define-color bg_active rgba(100, 100, 100, 0.5);
-
-/* base border color*/
-@define-color border_main rgba(255, 255, 255, 0.2);
-
-/* text color for entries, views and content in general */
-@define-color content_main white;
-/* text color for entries that are unselected */
-@define-color content_inactive rgba(255, 255, 255, 0.25);
+@define-color light #D8DEE9;
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@define-color" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)